plugins/cpu/cpu.quirk

```diff
@@ -1,3 +1,3 @@
 # Intel Atom Bay Trail [Silvermont]
-[DeviceInstanceId=CPUID\PRO_0&FAM_06&MOD_07]
-BcrAddr = 0x54
+[DeviceInstanceId=CPUID\PRO_0&FAM_06&MOD_37]
```

Comment:
I would think you may want to just have all of family 06 rather than this specific model.  GUIDs should be made for both

Reply:
Oh but wait, fam_06 actually hits big core too.  Never mind.